railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+ruby <%= "'#{RUBY_VERSION}'" %>
```

Comment:
Thanks @deivid-rodriguez , I already tried that and generating the app with `bundle exec rails new ~/my-test-app --dev` caused the following error parsing the Gemfile:

```
[!] There was an error parsing `Gemfile`: unexpected fraction part after numeric literal - ruby 2.3.1
        ^
/Users/alberto.almagro/my-test-app/Gemfile:3: syntax error, unexpected tIDENTIFIER, expecting end-of-input
git_source(:github) { |repo| "https://github.com/#{repo}.git" }
          ^. Bundler cannot continue.

 #  from /Users/alberto.almagro/my-test-app/Gemfile:2
 #  -------------------------------------------
 #  source 'https://rubygems.org'
 >  ruby 2.3.1
 #  git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 #  -------------------------------------------
```
I've surrounded it with single quotes, which is what is used in the line above for the source.